react/Banner.tsx

```diff
@@ -106,30 +103,4 @@ const Banner = (props: Props) => {
   )
 }
 
-Banner.propTypes = {
-  /** The description of the image */
-  description: PropTypes.string.isRequired,
-  /** Indicates if the route is external or not */
-  externalRoute: PropTypes.bool,
-  /** Max height size of the banner */
-  height: PropTypes.number.isRequired,
-  /** The image of the banner */
-  image: PropTypes.string.isRequired,
-  /** The mobile image of the banner */
-  mobileImage: PropTypes.string,
-  /** The page where the image is pointing to */
-  page: PropTypes.string,
-  /** Params of the url */
-  params: PropTypes.string,
-  /** The url where the image is pointing to, in case of external route */
-  url: PropTypes.string,
-  /** The url where the image is pointing to, in case of internal route */
-  customInternalURL: PropTypes.string,
-}
-
-Banner.defaultProps = {
-  height: 420,
-  description: '',
-}
```

Comment:
why do not use more `defaultProps`?

Reply:
I think using `defaultProps` like that hurts readability when using a function component since you can define them right when you destructure the `props` object. It's easier to read with less scrolling hahaha

Reply:
however if you want to add a default value for some props you always have to deconstruct the props params...

ref.: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md

Reply:
Yeah, but since we're always creating components with function and not classes and we always take the value of a prop as `const` I don't see when we have to take a prop from props more than once